src/core/text/Text.js

```diff
@@ -273,7 +273,10 @@ Text.prototype.updateText = function (respectDirty)
     if (navigator.isCocoonJS)
```

Comment:
It was an option as discussed https://github.com/pixijs/pixi.js/issues/2941 - My fault for suggesting the less pretty, but potentially faster option :)
I wish jsperf.com was still around to quickly benchmark a clearRect of 1x1 vs a clearRect of say 1920x1080!